kyuubi-server/src/main/scala/org/apache/kyuubi/engine/spark/SparkProcessBuilder.scala

```diff
@@ -337,9 +337,9 @@ object SparkProcessBuilder {
   final private[kyuubi] val sparkCoreScalaVersionRegex =
     """^spark-core_(\d\.\d+).*.jar$""".r
 
-  final private[kyuubi] val homeRegexForScala212 =
+  final private[kyuubi] lazy val SPARK_HOME_REGEX_SCALA_212 =
     """^spark-\d+\.\d+\.\d+-bin-hadoop\d+(\.\d+)?$""".r
 
-  final private[kyuubi] val homeRegexForScala213 =
+  final private[kyuubi] lazy val SPARK_HOME_REGEX_SCALA_213 =
```

Comment:
remove lazy, it have additional overhead

Reply:
fixed.